src/UglyToad.PdfPig.Tests/PublicApiScannerTests.cs

```diff
@@ -45,6 +45,7 @@ public void OnlyExposedApiIsPublic()
                 "UglyToad.PdfPig.Content.Page",
                 "UglyToad.PdfPig.Content.PageSize",
                 "UglyToad.PdfPig.Content.Word",
+                "UglyToad.PdfPig.Content.Line",
```

Comment:
It looks like this needs to be `TextLine` now to keep the API scanner test happy